pages/post/[postname].js

```diff
@@ -10,7 +10,19 @@ import Layout from '../../components/Layout';
 import Primary from '../../components/Primary';
 import CloudinaryImage from '../../components/CloudinaryImage';
 import Parser from '../../components/Parser';
-import { formatAuthors } from '../../components/BlogFeed';
+import TopicButton from '../../components/TopicButton';
+
+// todo: this currently exists here and in Post.js, time for an authors component?
+function formatAuthors(authors) {
```

Comment:
This looks like a duplicate function from the other file. Could we extract this into a helper?